src/main/java/frc/robot/subsystems/coraller/Angler.java

```diff
@@ -38,6 +40,18 @@ public Angler() {
         SparkBase.PersistMode.kPersistParameters);
     encoder = motor.getAbsoluteEncoder();
     pid = motor.getClosedLoopController();
+
+    SmartDashboard.putData(this);
+  }
+
+  @Override
+  public void initSendable(SendableBuilder builder) {
+    super.initSendable(builder);
+    builder.setSmartDashboardType(getName());
+    builder.setActuator(true);
+    builder.addDoubleProperty("SetPoint", this::getSetPoint, null);
```

Comment:
I think we are going to want a way to set the setpoint too.

```suggestion
    builder.addDoubleProperty("SetPoint", this::getSetPoint, this::changeSetPoint);
```